src/main/java/uk/gov/hmcts/reform/pcqbackend/exceptions/SchemaValidationException.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.pcqbackend.exceptions;
+
+
+/**
+ * This exception class is for capturing the JSON Schema validation.
```

Comment:
The strategy for Reform projects is to use SOLID / DRY / KISS pattern - in doing so the logic is that you don't need comments in code, I guess we can make a call specifically on PCQ?